src/main.cpp

```diff
@@ -53,10 +53,12 @@ void setup() {
 void loop(){
   if(not alarm->isArmed()){
     if(alarm->isButtonPressed()){
+      SerialComm::sendLogMessage("Alarm Armed");
       alarm->arm();
     }
   } else {
     if(alarm->isTripped()){
+      SerialComm::sendLogMessage("Alarm Tripped");
```

Comment:
Same as above. SerialComm::sendLogMessage() sends a packet and waits for the SHA1 hash response. Since nothing is listening when it sends the packet out, no response comes back and it times out.